### PR TITLE
Adds new OSD metrics

### PR DESCRIPTION
**Pull request by neurodrone**
- New metrics are added for tracking total no. of OSDs and their state for e.g. IN, UP, etc.
- The count of PGs being moved around is surfaced.
- Comments are added for all the health types.

cc: @cagedmantis @mdlayher @nickvanw

### CODE REVIEW

File: collectors/health.go [hunk 1]
```diff
@@ -27,16 +27,43 @@ import (
 // It surfaces changes in the ceph parameters unlike data usage that ClusterUsageCollector
 // does.
 type ClusterHealthCollector struct {
+	// conn holds connection to the Ceph cluster
 	conn Conn
 
-	DegradedPGs   prometheus.Gauge
-	UncleanPGs    prometheus.Gauge
+	// DegradedPGs shows the no. of PGs that have some of the replicas
```
Comment (mdlayher) [inline collectors/health.go]: Seems to be a bit of inconsistency in these comments between "no." and "count".  Mind picking one and using it throughout?  Apologies for the nitpick.

File: collectors/health.go [hunk 2]
```diff
@@ -87,6 +114,34 @@ func NewClusterHealthCollector(conn Conn) *ClusterHealthCollector {
...
+				Namespace: cephNamespace,
+				Name:      "osds",
+				Help:      "Count of total OSDs in the cluster",
+			},
+		),
+		RemappedPGs: prometheus.NewGauge(
+			prometheus.GaugeOpts{
+				Namespace: cephNamespace,
+				Name:      "pgs_remapped",
+				Help:      "No. of PGs that are remapped and incurring cluster-wide movement",
```
Comment (mdlayher) [inline collectors/health.go]: Same deal as above.

**Comment by cagedmantis** [comment]
LGTM

File: collectors/health.go (hunk 1 above)
Comment (neurodrone) [inline collectors/health.go]: Sure, if it helps going through it easier. :)

Comment (neurodrone) [inline collectors/health.go]: Addressed via https://github.com/digitalocean/ceph_exporter/commit/8091ea158a32819fef7968bf1afd2b34e5747313.

**Comment by mdlayher** [comment]
:ship:

**Pull request merged**